language/analysis/README.md

```diff
@@ -30,12 +30,13 @@ mvn clean compile assembly:single
 ```
 
 We can then run the assembled JAR file with the `java` command. The variable $COMMAND takes
-three values `entities`, `sentiment` or `syntax`.
+six values `entities-text`, `entities-file`, `sentiment-text`, `sentiment-file`,
+`syntax-text`, or `syntax-file`.
```

Comment:
That would moves us toward writing library code in the snippets, and the other 6 languages have separate snippets for raw strings vs gcs files, so Java would need to do the same to fit into the docs where the other snippets live.